### PR TITLE
Pusher

### PR DESCRIPTION
**Pull request by lcoderre**
What's new:
- Server creates a Stats schema
- Server starts services
  - Pusher monitor
- When the server gets something new (only Data is handled for now) from Pusher, it saves it in the DB.
  - You can see the contents updating in `http://localhost:3000/stats` by refreshing.

### CODE REVIEW

**Comment by simonbenoit** [comment]
Great work +1

File: app/controllers/index.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ var mongoose = require('mongoose');
```
Comment (jpgauthier) [inline app/controllers/index.js]: Let's drop that useless route soon?

File: app/controllers/pusher/data.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+exports.save = function(data) {
+	console.log("NINJA: Data received from device: " + data.D);
+
+	var o = new Stats({
+		// Ninja equivalents
+		guid: data.GUID,
+		data: data.DA,
+		deviceId: data.D,
+		vendorId: data.V,
+		portNumber: data.G,
```
Comment (jpgauthier) [inline app/controllers/pusher/data.js:15]: Not sure if we really need vendorId and portNumber, but why not...

File: app/tools/notifs.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+'use strict';
```
Comment (jpgauthier) [inline app/tools/notifs.js:1]: We'll discuss about this strategy tonight

Comment (lcoderre) [inline app/tools/notifs.js:1]: So I kept the notif file, just in case. But now it emits a create event on Stats schema when Pusher sends a stats.

**Comment by lcoderre** [comment]
@krapskill, or @bobbylavoie you can now listen to any received data this way. Might need a filtering instead of being notified for **every** data event, but it's in another iteration ;-)

``` javascript

var Stats = mongoose.model('Stats');
Stats.on('create', function(data) {
    console.log("Notified of incoming data");
    console.log(data);
});
```

The callback assumes a **data** which is actually the Stats object.

**Pull request merged**